ydb/core/scheme/scheme_tablecell.cpp

```diff
@@ -135,29 +139,35 @@ namespace {
 
         const char* buf = data.data();
         const char* bufEnd = data.data() + data.size();
-        if (bufEnd - buf < static_cast<ptrdiff_t>(sizeof(ui16)))
+        if (Y_UNLIKELY(bufEnd - buf < static_cast<ptrdiff_t>(sizeof(ui16))))
             return false;
 
         ui16 cellsSize = ReadUnaligned<ui16>(buf);
         buf += sizeof(cellsSize);
 
-        resultCells.resize(cellsSize);
+        resultCells.resize_uninitialized(cellsSize);
```

Comment:
I'm worried that `resize_uninitialized` is a non-standard vector extension, but I guess it's ok for now.